packages/office-addin-manifest/test/manifests/manifest.json

```diff
@@ -1,120 +1,82 @@
 {
   "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
-  "id": "df2fa15f-332a-47aa-87c2-e30c20686eee",
-  "manifestVersion": "m365DevPreview",
+  "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
   "version": "1.0.0",
+  "manifestVersion": "m365DevPreview",
   "name": {
-    "short": "My Office Add-in",
-    "full": "My Office Add-in"
+    "short": "Contoso Task Pane Add-in",
+    "full": "Contoso Task Pane Add-in"
   },
   "description": {
     "short": "A template to get started.",
-    "full": "A template to get started."
+    "full": "This is the template to get started."
   },
-  "icons": {
-    "outline": "test/assets/icon-32.png",
-    "color": "test/assets/icon-32.png"
-  },
-  "accentColor": "#FFFFFF",
   "developer": {
     "name": "Contoso",
     "websiteUrl": "https://www.contoso.com",
     "privacyUrl": "https://www.contoso.com/privacy",
-    "termsOfUseUrl": "https://www.contoso.com/terms"
+    "termsOfUseUrl": "https://www.contoso.com/servicesagreement"
+  },
+  "icons": {
+    "outline": "test/assets/icon-32.png",
+    "color": "test/assets/icon-32.png"
   },
+  "accentColor": "#230201",
   "localizationInfo": {
-    "defaultLanguageTag": "en-us"
+    "defaultLanguageTag": "en-us",
+    "additionalLanguages": [
+        {
+          "languageTag": "es-es",
+          "file": "locales/es-es.json"
+        }
+    ]
   },
-  "extension": {
-    "requirements": {
-      "scopes": ["Mailbox"],
-      "capabilities": [{ "name": "Mailbox", "minVersion": "1.3" }]
-    },
-    "runtimes": [
-      {
-        "id": "ShowTaskpane",
-        "type": "general",
-        "code": {
-          "page": "https://localhost:3000/taskpane.html"
-        },
-        "lifetime": "short",
-        "actions": [
-          {
-            "id": "ShowTaskpane.show",
-            "type": "contextual-launch",
-            "view": "",
-            "pinnable": false
-          }
-        ]
-      },
-      {
-        "id": "ExecuteFunction",
-        "type": "general",
-        "code": {
-          "page": "https://localhost:3000/commands.html",
-          "script": "https://localhost:3000/commands.js"
-        },
-        "lifetime": "short",
-        "actions": [
-          {
-            "id": "ExecuteFunction.action",
-            "type": "execution",
-            "name": "action"
-          }
-        ]
-      }
-    ],
-    "ribbons": [
```

Comment:
So we have no definition of the ribbon button anymore?

Reply:
I don't think these definitions are needed here, as this is just for testing for manifest manipulation.
We will need it in the task pane, as well some other definitions there, but they can get pretty lengthy and as we cannot test right now, I don't know which are necessary or not

Reply:
I would think we want the test manifest to match the taskpane manifest as much as possible in order to cover the same things.  There could be bugs in that wouldn't be caught by the test if they were a result of things in the taskpane manifest but not in the test manifest.

Reply:
I added all the sections that I thought would be relevant for the outlook test now.